src/natcap/invest/cli.py

```diff
@@ -465,6 +465,11 @@ def main(user_args=None):
             LOGGER.info('Imported target %s from %s',
                         model_module.__name__, model_module)
 
+            for arg_key, val in parsed_datastack.args.items():
+                if type(model_module.MODEL_SPEC.get_input(arg_key)) in {
+                        spec.RasterInput, spec.SingleBandRasterInput, spec.VectorInput}:
+                    parsed_datastack.args[arg_key] = utils._GDALPath.from_uri(val).to_string()
```

Comment:
Convert spatial paths to gdal-ready format. Eventually this should happen in a standardized preprocessing step at the beginning of model execution.